cypress/e2e/createEditTenure.cy.js

```diff
@@ -0,0 +1,406 @@
+
+            cy.url().should("contain", `tenure/${tenureId}/add`);
+            createTenurePage.addPropertyHeading().should('be.visible')
+            createTenurePage.propertyAddress().should('be.visible')
+
+            createTenurePage.tenureTypeSelection().select("Non-Secure")
+            createTenurePage.tenureStartDateDayContainer().clear().type("01")
+            createTenurePage.tenureStartDateMonthContainer().clear().type("01")
+            createTenurePage.tenureStartDateYearContainer().clear().type("2090")
+            cy.contains("Next").click();
+            createTenurePage.searchContainer().should('be.visible')
+            createTenurePage.searchButton().should('be.visible')
+            
+
+            //select resident to add to new tenure
+            const searchTerm = "brown"
+            createTenurePage.searchContainer().clear().type(searchTerm);
+            createTenurePage.searchButton().click();
+            createTenurePage.searchResults().contains(searchTerm.replace(/\*/g, ""), { matchCase: false });
+            cy.wait(10000) // Allows enough time to retrieve tenure with the latest version
+            for (let i = 0; i < 1; i++) {
+                createTenurePage.addAsNamedTenureHolderButton().eq(i).click()
+            }
+            createTenurePage.pageAnnouncementContainer().should('contain', 'Person added to tenure');
+
+            for (let i = 0; i < 1; i++) {
+                createTenurePage.addAsNamedTenureHolderButton().eq(i).click()
+            }
+            createTenurePage.pageAnnouncementContainer().should('be.visible')
+            createTenurePage.pageAnnouncementContainer().contains("The person is already added")
+
+            const searchTerm2 = "tre"
+            createTenurePage.searchContainer().clear().type(searchTerm2);
+            createTenurePage.searchButton().click();
+            createTenurePage.searchResults().contains(searchTerm2.replace(/\*/g, ""), { matchCase: false });
+            cy.wait(10000)
+            for (let i = 0; i < 1; i++) {
+                createTenurePage.addAsHouseholdMember().eq(i).click()
+            }
+            createTenurePage.pageAnnouncementContainer().should('contain', 'Person added to tenure');
+
+            for (let i = 0; i < 1; i++) {
+                createTenurePage.addAsHouseholdMember().eq(i).click()
+            }
+            createTenurePage.pageAnnouncementContainer().should('be.visible')
+            createTenurePage.pageAnnouncementContainer().contains("The person is already added")
+                  
+            //To fix: not able to get to the next card
+            // for (let i = 0; i < 5; i++) {
+            //     createTenurePage.addAsNamedTenureHolderButtonTemp().eq(i).click()
+            // }
+            // createTenurePage.pageAnnouncementContainer().contains("Max. tenure holders added")
+
+        }) 
+    })
+
+    it('should create a new tenure and cancel', ()=> {

```

Comment:
This is just testing the cancel button, which should be a unit test, I think this should be removed